### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by abhsag24**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1 +1 @@
-include UIP.py requirements.txt mac-requirements.txt linux-requirements.txt README.md
+include requirements.txt mac-requirements.txt linux-requirements.txt README.md
```
Comment (abhsag24) [inline MANIFEST.in]: newline?

Comment (nkprince007) [inline MANIFEST.in]: Nope, removed UIP.py, no longer required

**Comment by nemaniarjun** [comment]
Seems good to me :)

**Comment by nkprince007** [comment]
Seems okay to merge, bro @abhsag24 
Do I go ahead?

**Comment by abhsag24** [comment]
@nkprince007 did it for you :P

**Comment by nkprince007** [comment]
Thanks, bro @abhsag24

**Pull request merged**